docs/content/learn/tutorials/general/metadata.md

```diff
@@ -0,0 +1,828 @@
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+{{< highlight xml "linenos=table" >}}
+<catalog uuid="c3da6d1d-c20c-4c7c-ae73-4010167a186b">
+  <metadata>
+    <title>Example OSCAL Document</title>
+    <published>2021-01-01T00:00:00-5:00</published>
+    <last-modified>2021-01-05T00:00:00-5:00</last-modified>
+    <version>1.0.0<version>
+    <oscal-version>1.0.4</oscal-version>
+  </metadata>
+  …
+</catalog>
+{{< /highlight >}}
+{{% /tab %}}
+{{% tab %}}
+{{< highlight json "linenos=table" >}}
+{
+  "catalog": {
+    "uuid": "c3da6d1d-c20c-4c7c-ae73-4010167a186b"
+    "metadata": {
+      "title": "Example OSCAL Catalog",
+      "published": "2021-01-01T00:00:00-5:00",
+      "last-modified": "2021-01-05T00:00:00-5:00",
+      "version": "1.0.0",
+      "oscal-version": "1.0.4"
+    }
+  }
+}
+{{< /highlight >}}
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml "linenos=table" >}}
+---
+catalog: 
+  uuid: c3da6d1d-c20c-4c7c-ae73-4010167a186b
+  metadata: 
+    title: Example OSCAL Catalog
+    published: 2021-01-01T00:00:00-5:00
+    last-modified: 2021-01-05T00:00:00-5:00
+    version: 1.0.0
+    oscal-version: 1.0.4
+{{< /highlight >}}
+{{% /tab %}}
+{{% /tabs %}}
+
+However, the metadata section has several other optional but important fields which we will cover in the following sections.
+
+### Providing additional document identifiers
+
+OSCAL documents, by their nature, may often require updates of varying impact. It is important to track these updates in a way that can be automated and managed by a wide array of systems and users. To that end, we must cover the concepts of "Document Series" and "Document Series Instances".
+
+A "Document Series" consists of all formats, updates and versions of a given document. In more human terms, if a content author writes "Document 1 version 1", "Document 1 version 2", and "Document 1 version 3.4", we could say that the "Document Series" is simply "Document 1". Furthermore, if a document author produces PDF, OSCAL JSON, and other formats of the same information, it may be useful to identify the entire collection of formats and versions in a single "Document Series".
+
+A "Document Series Instance" is a discrete document instance that is part of a "Document Series". Using the above example, "Document 1 version 2 represented in OSCAL JSON" is a "Document Series Instance".
+
+In OSCAL we use a document identifier to identify that a document is a member of a "Document Series" and a document UUID to identify a "Document Series Instance".
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+{{< highlight xml "linenos=table" >}}
+<catalog uuid="c3da6d1d-c20c-4c7c-ae73-4010167a186b">
+  <metadata>
+    <title>Example OSCAL Document</title>
+    <published>2021-01-01T00:00:00-5:00</published>
+    <last-modified>2021-01-05T00:00:00-5:00</last-modified>
+    <version>1.0.0<version>
+    <oscal-version>1.0.4</oscal-version>
+    <document-id scheme="http://www.doi.org/">10.1000/182</document-id>
```

Comment:
I don't think this is a good example since it's actually a real DOI for a different digital object (or 'document series' if you like), as your browser shows if you paste `doi:10.1000/182`

A fake example might use NIST's DOI prefix (10.6028). We might say something about not assigning DOIs belonging to other organizations....

Reply:
Is there a reserved DOI number for examples? For example, the DNS domain "example.com" is reserved for this purpose.

Reply:
In https://www.crossref.org/blog/doi-like-strings-and-fake-dois/ the folks at Crossref talk about having created http://doi.org/10.5555/12345678 - a real DOI for a fake article in a fake journal (by a fake person at a real university).

Reply:
I am adding the following text to clarify the use of the `10.1000/182` DOI.

```
In the following example, we illustrate how to include a document identifier using the [Digital Object Identifier](https://www.doi.org/driven_by_DOI.html) (DOI) System using the identifier `10.1000/182`. This DOI points to the [DOI Handbook](https://www.doi.org/10.1000/182) and is used here as an example only. In a typical OSCAL use case, the referenced DOI will point to the actual OSCAL document instead.
```

While not perfect, this contextualizes the use of this DOI.

@wendellpiez Do you think this is sufficient?